pkg/plugins/runtime/k8s/controllers/namespace_controller.go

```diff
@@ -38,11 +36,9 @@ type NamespaceReconciler struct {
 	DefaultMeshTemplate     mesh_proto.Mesh
 }
 
-// Reconcile is in charge for two things:
-//   - create NetworkAttachmentDefinition if CNI enabled and namespace has label 'kuma.io/sidecar-injection: enabled'
-//   - create 'default' mesh for cluster
+// Reconcile is in charge for creating NetworkAttachmentDefinition if CNI enabled and namespace has label 'kuma.io/sidecar-injection: enabled'
```

Comment:
It wasn't clear to me when I wrote that comment a while ago, but probably it should be `in charge of`  
```suggestion
// Reconcile is in charge of creating NetworkAttachmentDefinition if CNI enabled and namespace has label 'kuma.io/sidecar-injection: enabled'
```